example-react-app/src/app/car-editor/CarEditor.tsx

```diff
@@ -126,6 +129,7 @@ const CarEditor = observer((props: EntityEditorProps<Car>) => {
         <Field
           entityName={ENTITY_NAME}
           propertyName="manufacturer"
+          disabled={isDisabledForm}
```

Comment:
I wonder if it would be better to just hide the whole form when the entity is deselected. Also potentially we might not need `resetEntityEditorForm` in this case, since unmounting the component will make it lose the state anyway.

Reply:
In Cuba's Master-detail template we have the ability to preview an entity's information before editing the entity. Form with disabled fields works like the preview component. Maybe we need to create a separate Preview component in the template.